src/index.html

```diff
@@ -10,7 +10,10 @@
 </head>
 <body>
   <app-root></app-root>
-
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
Are these required for the responsive tables to work? Judging from the Bootstrap docs, that's not the case. So I'd prefer not to include jQuery.

Reply:
Unfortunately that is required for the mobile navbar functionality.